mule-linter-core/src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/ComponentAttributeValueRule.groovy

```diff
@@ -9,7 +9,7 @@ import java.util.regex.Pattern
 
 class ComponentAttributeValueRule extends Rule {
 
-    static final String RULE_ID = 'COMPONENT_ATTRIBUTE_VALUE'
+    static final String RULE_ID = 'COMPONENT_REQUIRED_ATTRIBUTES'
```

Comment:
Generally, the rule id and rule class names are matching. I think class name should include plural `Attributes` - `ComponentAttributesValueRule`.